app/views/checkouts/_checkout.html.erb

```diff
@@ -36,6 +36,10 @@
         <dt class="col-5">Recalled on:</dt>
         <dd class="col-5"><%= l(checkout.recalled_date, format: :short) %></dd>
       <% end %>
+      <% if !checkout.overdue? %>
```

Comment:
Should we use `unless`? Or a new accessor on checkout that means not overdue?

Reply:
Went with an `unless` for now.